app/components/list-apps/component.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  sortBy: ['handle:asc'],
+
+  arrayChanged: function() {
+    Ember.run.once(this, 'checkModelArray');
+  }.observes('model.[]'),
+
```

Comment:
The current best practice would be to do this without the use of the function prototype extension:

```javascript
arrayChanged: Ember.observer('model.[]', function() {...})
```